packages/components/pie-toast/src/index.ts

```diff
@@ -137,7 +244,8 @@ export class PieToast extends RtlMixin(LitElement) implements ToastProps {
             _changedProperties.has('message') ||
             _changedProperties.has('isDismissible') ||
             _changedProperties.has('isMultiline') ||
-            _changedProperties.has('leadingAction')) {
+            _changedProperties.has('leadingAction') ||
+            _changedProperties.has('duration')) {
```

Comment:
Can you remind me why this is necessary? If we still need to do it, I think it's worth adding a comment as a reminder

Reply:
Without it the component ob storybook won't work properly since this piece of code make the component update if one of these properties changes.

Reply:
Please can you add a comment to the code explaining this?

Reply:
I am sorry I missed this comment. I will add in a following PR for sure. Thank you!